Jokes_Twitter_Bot.py

```diff
@@ -0,0 +1,31 @@
+import tweepy
+import joke
+
+# Define the API endpoint to fetch memes
+reddit_api_url = "https://www.reddit.com/r/memes/random.json?limit=1"
+
+# Define a user agent to mimic a web browser
+user_agent = (
+    "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+    "(KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36"
+)
+
+meme_data = joke.fetch_random_meme(reddit_api_url, user_agent)
+meme_url = joke.extract_meme_url(meme_data)

```

Comment:
Where is this **extract_meme_url** method? I didn't find it in **joke.py**; is it in some specific branch?

Reply:
It is there..look carefully

Reply:
Ahh... previously it was there but one contributor made it in less lines of code by removing those methods.